test/extended/prometheus/prometheus.go

```diff
@@ -33,6 +33,53 @@ import (
 	exutil "github.com/openshift/origin/test/extended/util"
 )
 
+var _ = g.Describe("[Feature:Prometheus][Late] Alerts", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc = exutil.NewCLIWithoutNamespace("prometheus")
+
+		url, bearerToken string
+	)
+	g.BeforeEach(func() {
+		var ok bool
+		url, bearerToken, ok = locatePrometheus(oc)
+		if !ok {
+			e2e.Skipf("Prometheus could not be located on this cluster, skipping prometheus test")
```

Comment:
Prometheus missing is an error condition, my PR changed this to Failf and this should too.

Reply:
Makes sense, done.